engine/xml-config/src/main/java/org/datacleaner/configuration/JaxbConfigurationReader.java

```diff
@@ -757,6 +758,11 @@ private Datastore createDatastore(String name, CassandraDatastoreType datastoreT
     private Datastore createDatastore(String name, ElasticSearchDatastoreType datastoreType) {
         final String clusterName = getStringVariable("clusterName", datastoreType.getClusterName());
         final String hostname = getStringVariable("hostname", datastoreType.getHostname());
+        final String username = getStringVariable("username", datastoreType.getUsername());
+        final String password = getPasswordVariable("password", datastoreType.getPassword());
+        final boolean ssl = getBooleanVariable("ssl", datastoreType.isSsl(), false);
+        final String keystorePath = getStringVariable("keystore-path", datastoreType.getKeystorePath());
+        final String keystorePassword = getPasswordVariable("keystore-password", datastoreType.getKeystorePassword());
```

Comment:
Eh maybe I said it wrong, but actually _here_ we would use camelCase :-P Reason is that this is referring to system-properties which has a different convention.